### DISCUSSION TITLE
Python libraries

### DISCUSSION THREAD

**Discussion by delioda79** [SaptakBhoumik/Peregrine#26] [General]
I just read about this language and what drove my interest is the similarity with python, of course. Not that Python is my favourite language but I use it for ML and the problem I have with using anything else than python is the lack of resources. we use Tensorflow, we use numpy etc.. Now is there any plan to support interoperability, somehow, with python librraies? Once teh language is ready I doubt we will have, otherwise, things like the above mentioned libraries/drivers available and supported for a new and not much used language.

**Comment by SaptakBhoumik** [comment]
@delioda79 We will also make a python to peregrine transcompiler which wont be very difficult as both of them have very similar syntax
